### ISSUE TITLE
Included module method are overridden 

### ISSUE THREAD

**Issue by cbeer** [ruby/ostruct#23]
In ostruct 0.3.2, we're seeing some surprising behavior for subclasses that include methods from other modules. Unlike ostruct 0.2, the included methods are now overridden with the ostruct accessors:

```
require 'ostruct'

module Bar
  def baz
    'from bar'
  end
end

class Foo < OpenStruct
  include Bar
end

Foo.new(baz: 'a').baz

# in ostruct 0.2.0:
# => 'from bar'

# in ostruct 0.3.2:
# => 'a'
```

But if the method is defined on the class itself, the method is not overridden:

```
class Quux < OpenStruct
  def baz
    'from quux'
  end
end

Quux.new(baz: 'a').baz

# in ostruct 0.2.0:
# => 'from quux'

# in ostruct 0.3.2:
# => 'from quux'
```

We're looking into some possible work-arounds for our usage, but I wanted to confirm that new behavior is working as expected. Thanks.

**Comment by marcandre** [comment]
Thanks for opening this issue.

Indeed, the way I wrote the check to see where a method is defined compared to OpenStruct is incorrect. If you modify your example above with `Bar` being a class instead of a module it works as you expect it.

I will fix this shortly.

**Comment by cbeer** [comment]
Thanks for the quick fix!

**Comment by marcandre** [comment]
Sure thing. I'll waiting on the main CI and will then make a release

**Comment by marcandre** [comment]
Released in v0.3.3
